folium/plugins/draw.py

```diff
@@ -60,6 +60,29 @@ class Draw(JSCSSMixin, MacroElement):
 
     _template = Template(
         """
+        {% macro html(this, kwargs) %}
+            {% if this.export %}
+            <style>
```

Comment:
this style element was originally added to the header. Here it's in the body. I guess it doesn't make a difference really, does it?